package.json

```diff
@@ -4,7 +4,11 @@
   "version": "1.0.0",
   "author": "ECV Digital Bordeaux 16/17",
   "scripts": {
-    "dev": "webpack-dev-server --open --inline --hot"
+    "dev": "webpack-dev-server --open --inline --hot",
+    "lint:vue": "eslint src --ext .vue",
+    "lint:js": "eslint src --ext .js",
```

Comment:
Please add an `npm run lint` that does both.
Take a look at https://www.npmjs.com/package/npm-run-all